src/content/docs/en/basics/astro-components.mdx

```diff
@@ -318,6 +318,8 @@ const { title } = Astro.props;
 </div>
 ```
 
+Astro will pass an empty slot when a slot element exists but has no content to pass. Fallback content cannot be used as a default when an empty slot is passed. Fallback content is only displayed when no slot element can be found.
```

Comment:
```suggestion
Fallback content will only be displayed when no matching elements with the `slot="name"` attribute are found to pass into a named slot. 

Astro will pass an empty slot when a slot element exists but has no content to pass. Fallback content cannot be used as a default when an empty slot is passed. Fallback content is only displayed when no slot element can be found.
```
Do we think leaving in that first sentence is too much?

Reply:
I don't think it's "passed IN AS a named slot?" (It is not the named slot itself, that's the destination?) 

I would still say passed (in) TO a named slot? Are you 100% on that?

But yes, we are very close!  :joy:

Reply:
Y'know what, you're right! You pass things _into_ a slot!

So then the final™️ wording:

---

Fallback content will only be displayed when there are no matching elements with the slot="name" attribute being passed in to a named slot.

Astro will pass an empty slot when a slot element exists but has no content to pass. Fallback content cannot be used as a default when an empty slot is passed. Fallback content is only displayed when no slot element can be found.

Reply:
final-1(1)-a-2 final!